### ISSUE TITLE
Consider onboarding post installation

### ISSUE THREAD

**Issue by ChrisChinchilla** [errata-ai/vale-vscode#43]
As mentioned in #42 and also in several user testing interviews, the initial experience of the extension can be confusing. Perhaps a first step MVP is a post install pop up like many other extensions have.

**Comment by osg** [comment]
Yes. I have put my usage on hold until I can get help.

**Comment by osg-grafana** [comment]
Yes. I have put my usage on hold until I can get help internally.

**Comment by Stanzilla** [comment]
Yeah it is not very clear if you have to set the CLI path to an absolute value or if it can just use what is on PATH. For me it did not work with vale on the PATH in WSL.
